plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/commit/CommitPresenter.java

```diff
@@ -236,10 +245,33 @@ private void push(Path location) {
             })
         .catchError(
             error -> {
-              notificationManager.notify(constant.pushFail(), FAIL, FLOAT_MODE);
+                final String providerName = GitUtil.getProviderNameFromError(error);
+                if (!StringUtils.isNullOrEmpty(providerName)) {
```

Comment:
I think it is better to use static import for `isNullOrEmpty()` method

Reply:
Ok